lib/appsignal.ex

```diff
@@ -0,0 +1,13 @@
+defmodule Helix.Appsignal do
+
+  # HACK: __using__ is expanded at compile-time. Since Appsignal is not
```

Comment:
Well, i know that there is a hack here (and everywhere else; after all, the game is called **HACKER** Experience for a reason :joy:)